fms_fsdp/utils/checkpointing_utils.py

```diff
@@ -168,7 +172,14 @@ def _validate_ckp_path(self, path):
         return None
 
     def load(
-        self, model, optimizer, dataloader, path="", reset_stepcount=False, strict=True
+        self,
+        model,
+        optimizer,
+        dataloader,
+        path="",
+        reset_stepcount=False,
+        strict=True,
+        **kwargs,
```

Comment:
might want to just add an `is_compiled=False` parameter to this load. I think it complicates it being a kwarg